### ISSUE TITLE
Quick fix for this shortcut key issue

### ISSUE THREAD

**Issue by ZeroHawkeye** [zed-industries/zed#14996]
### Check for existing issues

- [X] Completed

### Describe the feature

Unable to select the currently selected action by pressing Enter or Tab after pressing the shortcut key (ctrl->)

### If applicable, add mockups / screenshots to help present your vision of the feature

![image](https://github.com/user-attachments/assets/9f7e7e6a-c3ca-4402-a4c2-63c0dce1d79a)

**Comment by JosephTLyons** [comment]
Hey @ZeroHawkeye, what OS are you on here?

**Comment by ZeroHawkeye** [comment]
arch Linux
kde plasma 6 desktop

**Comment by ZeroHawkeye** [comment]
And I can't seem to modify this quick fix configuration item in the shortcut key settings.

**Comment by notpeter** [comment]
I am not able to reproduce this.
@ZeroHawkeye Are you still seeing this?

**Comment by JosephTLyons** [comment]
I'm going to close this issue out, due to inactivity. @ZeroHawkeye, please feel free to reopen this if you are still running into this!
